shard.lock

```diff
@@ -0,0 +1,6 @@
+version: 1.0
```

Comment:
Don't commit this file either.  Bindgen is in a "funny" spot where it's both a library and an application.  For now I chose to not commit the lock file because of that.